ddtrace/tracer/context.go

```diff
@@ -38,7 +38,12 @@ func SpanFromContext(ctx context.Context) (Span, bool) {
 // StartSpanFromContext returns a new span with the given operation name and options. If a span
 // is found in the context, it will be used as the parent of the resulting span. If the ChildOf
 // option is passed, the span from context will take precedence over it as the parent span.
+// If ctx is nil, this will use context.Background().
```

Comment:
This is nothing but a safety mechanism, I don't think it warrants documentation changes.